src/listening_groups.py

```diff
@@ -0,0 +1,61 @@
+import csv
+import os.path
+
+from core_data_modules.logging import Logger
+from core_data_modules.traced_data import Metadata
+import time
+
+from src.lib import PipelineConfiguration
+
+log = Logger(__name__)
+
+class ListeningGroup(object):
+
+    @classmethod
+    def tag_listening_groups_participants(cls, user, data, listening_group_dir):
+        '''
+        This tags uids who participated in repeat listening groups and/or weekly listening
+        group sessions.
+        :param user: Identifier of the user running this program, for TracedData Metadata.
+        :type user: str
+        :param data: TracedData objects to tag listening group participation to.
+        :type data: iterable of TracedData
+        :param listening_group_dir: Directory containing de-identified listening groups contacts CSVs.
```

Comment:
Could you add a brief description of what the format for these is?

Reply:
updated thanks!